colormap-renderer/src/main/java/com/esri/arcgisruntime/sample/colormaprenderer/MainActivity.java

```diff
@@ -0,0 +1,152 @@
+import com.esri.arcgisruntime.raster.ColormapRenderer;
+import com.esri.arcgisruntime.raster.Raster;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A sample class which demonstrates the ColorMapRenderer.
+ */
+public class MainActivity extends AppCompatActivity {
+
+  private MapView mMapView;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+    // retrieve the MapView from layout
+    mMapView = (MapView) findViewById(R.id.mapView);
+    // define permission to request
+    String[] reqPermission = new String[] { Manifest.permission.WRITE_EXTERNAL_STORAGE };
+    int requestCode = 2;
+    // For API level 23+ request permission at runtime
+    if (ContextCompat.checkSelfPermission(MainActivity.this,
+        reqPermission[0]) == PackageManager.PERMISSION_GRANTED) {
+      loadRaster();
+    } else {
+      // request permission
+      ActivityCompat.requestPermissions(MainActivity.this, reqPermission, requestCode);
+    }
+  }
+
+  /**
+   * Using values stored in strings.xml, builds path to ShastaBW.tif.
+   *
+   * @return the path to raster file
+   */
+  private String buildRasterPath() {
+    // get sdcard resource name
+    File extStorDir = Environment.getExternalStorageDirectory();
+    // get the directory
+    String extSDCardDirName = this.getResources().getString(R.string.config_data_sdcard_offline_dir);
+    // get raster filename
+    String filename = this.getString(R.string.config_raster_name);
+    // create the full path to the raster file
+    return extStorDir.getAbsolutePath()
+        + File.separator
+        + extSDCardDirName
+        + File.separator
+        + filename
+        + ".tif";
+  }
+
+  /**
+   * Loads ShastaBW.tif as a Raster and adds it to a new RasterLayer. RasterLayer is then added to the map as an
+   * operational layer. A List of color values is created (0-149: red) (150-250: yellow). The List is passed to a new
+   * ColorMapRenderer, which is then set to the RasterLayer Rendererer. Map viewpoint is then set based on Raster
+   * geometry.
+   */
+  private void loadRaster() {
+    // create a raster from a local raster file
+    Raster raster = new Raster(buildRasterPath());
+    // create a raster layer
+    final RasterLayer rasterLayer = new RasterLayer(raster);
+    // create a Map with imagery basemap
+    ArcGISMap map = new ArcGISMap(Basemap.createImagery());
+    // add the map to a map view
+    mMapView.setMap(map);
+    // add the raster as an operational layer
+    map.getOperationalLayers().add(rasterLayer);
+    // create a color map where values 0-149 are red (0xFFFF0000) and 150-250 are yellow (0xFFFFFF00)
+    List<Integer> colors = new ArrayList<>();
+    for (int i = 0; i <= 250; i++) {
+      if (i < 150) {
+        colors.add(i, 0xFFFF0000);
+      } else {
+        colors.add(i, 0xFFFFFF00);
```

Comment:
Replace sRGB values with Color constants, e.g. `Color.RED`, and update the comment as appropriate.